src/ui/handler/scroll_zoom.js

```diff
@@ -79,15 +79,15 @@ class ScrollZoomHandler {
 
     /**
      * Set the zoom rate of a trackpad
-     * @param {number} [zoomRate = 1/100]
+     * @param {number} [zoomRate=1/100] Zoom rate in some unit.
```

Comment:
I'm not sure what this unit is (same for wheelRateZoom)

Reply:
Theres a lot of magic functions and numbers this goes through, its hard to really describe a unit.
maybe something like
`The rate used to scale the mouse wheel deltaY to a zoom value`